ods_ci/tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -92,26 +98,40 @@ Select Container Size
     Click Element  xpath:${JUPYTERHUB_DROPDOWN_XPATH}\[1]
     Click Element  xpath://span[.="${container_size}"]/../..
 
-Wait Until GPU Dropdown Exists
-    [Documentation]    Verifies that the dropdown to select the no. of GPUs exists
-    Wait Until Page Contains    Number of GPUs
+Wait Until Accelerator Dropdown Exists
+    [Documentation]    Verifies that the dropdown to select the Accelerator exists
     Page Should Not Contain    All GPUs are currently in use, try again later.
-    Wait Until Page Contains Element    xpath:${KFNBC_GPU_DROPDOWN_XPATH}
-    ...    error=GPU selector is not present in JupyterHub Spawner
+    Wait Until Page Contains Element    xpath:${KFNBC_ACCELERATOR_DROPDOWN_XPATH}
+    ...    error=Accelerator selector is not present in JupyterHub Spawner
 
-Set Number Of Required GPUs
-    [Documentation]  Sets the gpu count based on the ${gpus} argument
+Set NVidia GPU Accelerator
+    [Documentation]  Set NVidia GPU Accelerator
+    Click Element  xpath:${KFNBC_ACCELERATOR_DROPDOWN_XPATH}
+    Click Element  xpath:${KFNBC_ACCELERATOR_DROPDOWN_NVIDIA_XPATH}
+
+Set Number Of Required Accelerators
+    [Documentation]  Sets the Accelerators count based on the ${gpus} argument
     [Arguments]  ${gpus}
```

Comment:
you're changing the variables and keywords name from GPU to Accelerator. Maybe you can change the argument here as well

Reply:
Fixed in [cbf6eec](https://github.com/red-hat-data-services/ods-ci/pull/971/commits/cbf6eec202c7ac338bdaeea757e0597f81b7df61)